config.json

```diff
@@ -642,9 +642,12 @@
       "slug": "diffie-hellman",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 5,
       "topics": [
-
+        "Cryptography",
+        "Integers",
+        "Algorithms",
+        "Transforming"
```

Comment:
Since the big.Int is a central part of the solution, I would suggest Interfaces. Not in the Go "interface" sense, but in the "make use an existing interface/package" sense.  What do you think ?